erpnext/controllers/accounts_controller.py

```diff
@@ -1351,3 +1351,11 @@ def validate_quantity(child_item, d):
 @erpnext.allow_regional
 def validate_regional(doc):
 	pass
+
+@frappe.whitelist()
+def get_shipping_rule_account_head(master_doctype, master_name):
+	if not master_name:
+		return
+	from frappe.model import default_fields
+	ship_rule = frappe.get_doc(master_doctype, master_name)	
+	return ship_rule
```

Comment:
@shantanu-kadam-epi  if this function does not need the accounts controller function then it should be added to the accounts utility file.  ./apps/erpnext/erpnext/accounts/utils.py
```suggestion

@frappe.whitelist()
def get_shipping_rule_account_head(doctype, docname):
	if not docname:
		return
	from frappe.model import default_fields
	shipping_rule = frappe.get_doc(doctype, docname)	
	return shipping_rule
```